Schemata/sarif-schema.json

```diff
@@ -1267,6 +1267,12 @@
           "type": "string"
         },
 
+        "occurrenceCount": {
+          "description": "A positive integer specifying the number of times a result with this result object's correlationGuid has been observed",
```

Comment:
>result object's correlationGuid has been observed [](start = 95, length = 49)

btw - the correlationGuid may not be expressed in all cases where we use occurrenceCount., so we should prefer simply defining this thing in terms of what a correlationGuid represents. Something like so:

A positive integer specifying the number of times this logically unique result has been observed in this run.

Reply:
That is better. I'll change it in my current PR (I'm working on fixing the rogue `DictionaryHint`s).